discord/raw_models.py

```diff
@@ -127,20 +127,29 @@ class RawReactionActionEvent(_RawReprMixin):
         The guild ID where the reaction got added or removed, if applicable.
     emoji: :class:`PartialEmoji`
         The custom or unicode emoji being used.
+    member: Optional[:class:`Member`]
+        The member who added the reaction. 
```

Comment:
```suggestion
        The member who added the reaction. Only available if `event_type` is `REACTION_ADD`.
```

I prefer this to the note block below.